pulp_rpm/app/viewsets.py

```diff
@@ -301,6 +316,44 @@ class RpmRepositoryVersionViewSet(RepositoryVersionViewSet):
 
     parent_viewset = RpmRepositoryViewSet
 
+    DEFAULT_ACCESS_POLICY = {
+        "statements": [
+            {
+                "action": ["list", "retrieve"],
+                "principal": "authenticated",
+                "effect": "allow",
+                "condition": "has_repository_model_or_obj_perms:rpm.view_rpmrepository",
+            },
+            {
+                "action": ["destroy"],
+                "principal": "authenticated",
+                "effect": "allow",
+                "condition": [
+                    "has_repository_model_or_obj_perms:rpm.delete_rpmrepository_versions",
+                    "has_repository_model_or_obj_perms:rpm.view_rpmrepository",
+                ],
+            },
+            {
+                "action": ["destroy"],
+                "principal": "authenticated",
+                "effect": "allow",
+                "condition": [
+                    "has_repository_model_or_obj_perms:rpm.delete_rpmrepository",
+                    "has_repository_model_or_obj_perms:rpm.view_rpmrepository",
+                ],
+            },
+            {
+                "action": ["repair"],
+                "principal": "authenticated",
+                "effect": "allow",
+                "condition": [
+                    "has_repository_model_or_obj_perms:rpm.sync_rpmrepository",
```

Comment:
you can also introduce a specific permission for repiar instead of re-using the sync one